src/authentication/create-account-details/index.tsx

```diff
@@ -52,14 +54,20 @@ export class CreateAccountDetails extends React.Component<Properties, State> {
     return this.props.errors.image;
   }
 
+  renderImageUploadIcon = (): JSX.Element => <IconUpload2 isFilled size={24} color='#EDEDEF' />;
```

Comment:
Do we need to use the `color` prop here or can we just style the upload component with the right color? They're all the same.

Reply:
Yes you're right @dalefukami. It looks to me that the global default colour is the same as what is being set here (`#EDEDEF`) , therefore I have removed the `color` prop. Also, the view box for Icons is 24x24 on each icon in zUI, as a result, I have removed the `size` prop as well as `24` is essentially the default height and width. Let me know if that doesn't seem right to you. Thanks!